lib/plugins/features/movies/controller.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const Movie = require('../../../models/movie');
+
+// defining code that is shared accross different files. Important to include so that the funciton or the variable can be used outside of this file
+exports.create = (payload) => {
+  // saves the apyload as the movie object in the database
+  return new Movie().save(payload).then( (movie) => {
```

Comment:
Nit: Generally we define `then`s on newlines. So it would look something like this:
```js
return new Movie().save(payload)
.then((movie) => {
  // do something with movie
});
```